entity-management-definitions/src/main/java/eu/europeana/entitymanagement/normalization/EntityFieldsCleaner.java

```diff
@@ -69,7 +77,18 @@ public void cleanAndNormalize(Entity entity) {
 	}
     }
 
-    @SuppressWarnings({ "unchecked", "rawtypes" })
+	private WebResource normalizeWebResource(WebResource existingFieldValue) {
+		WebResource webResource = new WebResource(existingFieldValue);
+		// generate thumbnail if missing
+		if(StringUtils.isEmpty(webResource.getThumbnail())){
+			webResource.setThumbnail(thumbnailBaseUrl + URLEncoder.encode(existingFieldValue.getId(),
```

Comment:
the Thumbnails should be generated only for Europeana objects, the generated ThumbnailUrls will not work on external images, like the ones from Wikimedia Commons (which may occur in the depiction field, but not in isShownBy)

Reply:
Thanks @gsergiu. Now fixed.